src/libraries/Microsoft.PowerFx.Core/Public/Types/FormulaType.cs

```diff
@@ -70,7 +71,25 @@ public abstract class FormulaType
         internal FormulaType(DType type)
         {
             _type = type;
-        }
+        }
+
+        // primitive types currently supported by UDFs
+        public static readonly IEnumerable<KeyValuePair<DName, FormulaType>> _primitiveTypes = new Dictionary<DName, FormulaType>()

```

Comment:
>public [](http://example.com/codeflow?start=8&length=6)

private?

Reply:
I am using this is PA client . may be i should drop _ ? or i could create a copy of this in client , but i thought having in one place is better